mindocr/data/transforms/east_transforms.py

```diff
@@ -0,0 +1,429 @@
+import math
+import numpy as np
+import cv2
+from PIL import Image
+from shapely.geometry import Polygon
+
+__all__ = ['EASTProcessTrain']
+
+
+def cal_distance(x1, y1, x2, y2):
```

Comment:
Some potential refactoring on these functions to make them more clear and readable.
- Functions that are only called by other functions, i.e., internal use, can be defined as 'private' function like `_func()`
- Seems all functions outside the class can be defined inside the class.

Reply:
Modified.